mypyc/test-data/run-bytes.test

```diff
@@ -24,6 +24,24 @@ try:
 except TypeError:
     pass
 
+[case testBytesInit]
+def test_bytes_init() -> None:
+    b1 = bytes([5])
+    assert b1 == b'\x05'
+    b2 = bytes([5, 10, 12])
+    assert b2 == b'\x05\n\x0c'
+    b3 = bytes(bytearray(b'foo'))
+    assert b3 == b'foo'
+    b4 = bytes(b'aaa')
+    assert b4 == b'aaa'
+    # b5 = bytes(5)
+    # assert b5 == b'\x00\x00\x00\x00\x00'
```

Comment:
This line causes a runtime error

Reply:
Unfortunately this means that C API function is not a 1:1 replacement for `bytes(x)` and can't be used as such. We now have these options: 

1. Don't add the primitive
2. Add an argument type restriction to the use of the primitive (e.g. only support `bytes` or `list` -- whatever works).
3. Implement a C helper that implements the missing functionality and falls back to the C API function for the supported functionality.

What do you think? Also test `bytearray(5)`.

Reply:
`bytearray(5)` works well